src/LasagnaController.h

```diff
@@ -42,21 +43,23 @@ class LasagnaController
   } Input;
 
   typedef struct __attribute__((packed)) {
-    float freq                 =   20;        // control freqency
-    float k_v                  =   1*.375e-3;      // velocity gain
-    float k_h                  =   1*.375e-3;    // altitude gain
-    float b_dldt               =   0.000666;    // balast dl/dt (kg/s)
-    float v_dldt_a             =   0;
-    float v_dldt_b             =   0.0030;    // valve dl/dt (kg/s))
-    float b_tmin               =   2;         // minimum ballast event time
-    float v_tmin               =   3;         // minimum valve event time
-    float h_cmd                =   13500;     // altidute comand
-    float kfuse                =   6;
-    float kfuse_val            =   0.75;
-    float ss_error_thresh      =   750;
-    float v_limit              =   0.5;
-    float equil_h_thresh       =   10000;      //altitude where controller transitions to normal mode
-    float launch_h_thresh      =   300;
+    float freq                 =   20;          // Control Freqency (Hz)
+    float gain                 =   0.14/1000;   // Total gain magnititude (g / m)
+    float damping              =   1.2;         // damping ratio (unitless)
+    float v_gain               =   0;           // velocity gain (g/s / m/s) 
+    float h_gain               =   0;           // altitude gain (m/s / km)
+    float bal_dldt             =   0.666;       // balast dl/dt (g / s)
+    float val_dldt_a           =   0;           
```

Comment:
What's the difference between `val_dldt_a` and `val_dldt_b`?

Reply:
it's so that it can be function of overpresssure. We aren't using it now but might in the future